src/index.html

```diff
@@ -11,6 +11,50 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      class="form"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post"
+      data-qa="big"
+    >
+
+      <img 
+        class="image"
+        src="images/Search.svg" 
+        alt="search"
+      >
+
+      <input 
+          class="input" 
+          type="text" 
+          name="search" 
+          data-qa="keypress" 
+          placeholder= "Try &quot;Los Angeles&quot;" 
+      >
```

Comment:
```suggestion
      <input 
        class="input" 
        type="text" 
        name="search" 
        data-qa="keypress" 
        placeholder= "Try &quot;Los Angeles&quot;" 
      >
```

Reply:
Should be 2 spaces

Reply:
fix it 
<img width="502" alt="image" src="https://user-images.githubusercontent.com/53611910/201547924-e1f66f8b-6853-460d-89d0-e5e1c500840b.png">